announcements/2020-Week-44/announcement.md

```diff
@@ -0,0 +1,17 @@
+# :zap: This Week in Provider-Independent Announcements (October 16, 2020):zap:
+
+## Giant Swarm Catalog (Managed Apps)
+
+1. [Cert Manager v 2.3.1](https://github.com/giantswarm/cert-manager-app/blob/master/CHANGELOG.md#231---2020-10-29) removes deprecated APIs and upgrades to upstream v1.0.3.
+2. [Kong v0.9.2](https://github.com/giantswarm/kong-app/blob/release-v0.9.x/CHANGELOG.md#092---2020-10-28) adds minReadySeconds (optional parameter) to the deployment template. This helps avoid downtime during rolling updates.
+
```

Comment:
```suggestion
3. [EFK v0.3.4](https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#034---2020-10-30) improves reliability of upgrades. It enables PodDisruptionBudget and builds are now automatically tested with functional testing.
```

Reply:
It seems I can only comment a suggestion, not edit it, so:
- I would add info that pod anti-affinity is now enabled by default
- I would add: Please read <https://github.com/giantswarm/efk-stack-app/tree/v0.3.4#important-notes---read-before-you-deploy>

Reply:
3. [EFK v0.3.4](https://github.com/giantswarm/efk-stack-app/blob/master/CHANGELOG.md#034---2020-10-30) improves reliability of upgrades. Builds now automatically do functional testing. It also enables pod anti-affinity and PodDisruptionBudget by default. Please [read before you deploy](https://github.com/giantswarm/efk-stack-app/tree/v0.3.4#important-notes---read-before-you-deploy).